UofS-Pulse-Binfo/analyzedphenotypes
# SIO Study Tripal WS compatible

Field supports Tripal Web Services.



This PR made an update to hook_bundle_field_info() and to the field's elementInfo() and load().
The former will allow this field to support multiple values returned for both end user and Tripal Web Services by changing the cardinality settings, while the latter to alter the array structure and to define element definition.

## Metadata

 - Issue # https://github.com/UofS-Pulse-Binfo/analyzedphenotypes/issues/108 - Make SIO:Study Field Tripal Web Services Compatible



    https://github.com/tripal/tripal/issues/1010 Web Services - Invalid Content Type
    in the meantime, just replace any occurrence of + (plus sign) in @id URL seen in the address field of your browser.

Documentation:

## Description



## Dependencies



## Testing?



1. Delete sio__study if it is used in Trait content type.
2. Update sio__study and analyzedphenotypes.field.inc
3. In the Trait Content Type, Click manage fields then Check for new fields.
4. Add sio__study field to Trait Content type, Make sure to move this field out of disabled fields.
5. Access Traits Content Type in Tripal WS.
